wurst/event/OnUnitEnterLeave.wurst

```diff
@@ -37,26 +37,10 @@ constant ABILITY_ID = compiletime(ABIL_ID_GEN.next())
 constant eventTrigger = CreateTrigger()
 unit tempUnit = null
 let preplacedUnits = CreateGroup()
-constant trigger preplacedUnitsCall = CreateTrigger()..addAction(function initPreplacedUnits)
 
 function prepareUnit(unit u)
 	u..addAbility(ABILITY_ID)..makeAbilityPermanent(ABILITY_ID, true)
```

Comment:
move `eventTrigger.evaluate()`  here

Reply:
What do you mean?

Reply:
eventTrigger.evaluate() is called after each time you use prepareUnit.
So move it inside the function.

Reply:
Oh yeah you're right. Done.